src/snap.py

```diff
@@ -0,0 +1,101 @@
+import os
+import sys
+import pandas as pd
+from typing import Any, Dict, Tuple, List
+sys.path.append(os.path.abspath(''))
+from src.census_response import county_fips  # noqa: E402
+
+
+def load_excel(src: str) -> Any:
+    table = pd.read_excel(src, sheet_name=None, engine='openpyxl')
```

Comment:
is table a pd.DataFrame or some other type? Can you make the return type hint any more specific?